tests/test_dte_data_models.py

```diff
@@ -98,9 +125,58 @@ def setUp(self) -> None:
             monto_total=2996301,
         )
 
-    def test_init_fail(self) -> None:
-        # TODO: implement for 'DteDataL1()'
-        pass
+    def test_is_ok_negative_monto_total_in_tipo_dte_liquidacion_factura(self) -> None:
+        _ = dataclasses.replace(
+            self.dte_l1_1,
+            tipo_dte=TipoDteEnum.LIQUIDACION_FACTURA_ELECTRONICA,
+            monto_total=-1,
+        )
```

Comment:
If what you want here is for the test to fail **if** a validation exception is raised, then you need to change the test so that the exception is handled because otherwise you'll get a test **_error_** :-1: instead of a test **_failure_** :+1:.

**Comparison**:

:frowning_face: 

```
======================================================================
ERROR: test_is_ok_negative_monto_total_in_tipo_dte_liq... (tests.test_dte_data_models.DteDataL1Test)
----------------------------------------------------------------------

[...]

----------------------------------------------------------------------
Ran 338 tests in 0.191s

FAILED (errors=1, skipped=2)
Test failed: <unittest.runner.TextTestResult run=338 errors=1 failures=0>
```

vs.

:smile:

```
======================================================================
FAIL: test_is_ok_negative_monto_total_in_tipo_dte_liq... (tests.test_dte_data_models.DteDataL1Test)
----------------------------------------------------------------------

[...]

----------------------------------------------------------------------
Ran 338 tests in 0.184s

FAILED (failures=1, skipped=2)
Test failed: <unittest.runner.TextTestResult run=338 errors=0 failures=1>
```


```suggestion
        try:
            _ = dataclasses.replace(
                self.dte_l1_1,
                tipo_dte=TipoDteEnum.LIQUIDACION_FACTURA_ELECTRONICA,
                monto_total=-1,
            )
        except pydantic.ValidationError as exc:
            self.fail(f'{exc.__class__.__name__} raised')
```